Dockerfile.aarch64

```diff
@@ -39,6 +39,7 @@ RUN apt-get update && apt-get install -y \
 	libcap-dev \
 	libltdl-dev \
 	libsystemd-dev \
+	libyaml-dev \
```

Comment:
just wondering, do you need both this and yamllint for the swagger stuff on aarch64? I'm just wondering because it isn't in the other dockerfiles.